### PR TITLE
Add translated Chinese language docs for SharedMergeTree

### PR DESCRIPTION
**Pull request by justindeguzman**
### Changelog category (leave one):

- Documentation (changelog entry is not required)

Reviewed by @bianpengyuan, @tracywangnan, and @incfly.

There's still two lines that need to be translated, it'd be helpful to get translations for these:

> This will create the table `my_table` using the SharedMergeTree table engine.

> You don’t need to specify `ENGINE=MergeTree` as `default_table_engine=MergeTree` in ClickHouse Cloud. The following query is identical to the query above.

### CODE REVIEW

**Comment by robot-ch-test-poll** [comment]
This is an automated comment for commit 5aac62713715a37ad34548cad1bd0674fb98f4c2 with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/53624/5aac62713715a37ad34548cad1bd0674fb98f4c2/ci_running.html)
The overall status of the commit is 🔴 failure

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td>🔴 failure</td></tr>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td>🟢 success</td></tr>
</table>

**Comment by tracywangnan** [comment]
This will create the table my_table using the SharedMergeTree table engine.
通过使用 SharedMergeTree 表引擎可以创建 my_table 表
You don’t need to specify ENGINE=MergeTree as default_table_engine=MergeTree in ClickHouse Cloud. The following query is identical to the query above.
在ClickHouse Cloud中，由于default_table_engine=MergeTree，你不必再特别设置ENGINE=MergeTree。下面的查询语句和上面的完全一样。

File: docs/zh/engines/table-engines/mergetree-family/sharedmergetree.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+对于支持SharedMergeTree表引擎的实例，您不需要做任何额外变更。您可以像以前一样创建表，它会自动使用基于SharedMergeTree的表引擎，该引擎与您在CREATE TABLE查询中指定的引擎相对应。
+
+通过使用 SharedMergeTree 表引擎可以创建 my_table 表。
+
+```sql
+CREATE TABLE my_table(
+ key UInt64,
+ value String
+)
+ENGINE =MergeTree
```
Comment (hanfei1991) [inline docs/zh/engines/table-engines/mergetree-family/sharedmergetree.md:55]:
```suggestion
ENGINE = MergeTree
```

File: docs/zh/engines/table-engines/mergetree-family/sharedmergetree.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+正如您所见，尽管存储在ReplicatedMergeTree中的数据位于对象存储中，但元数据仍存储在每个clickhouse-server上。这意味着对于每个复制操作，元数据也需要在所有副本上进行复制。
+
+![ReplicatedMergeTree Diagram](../../../images/shared-merge-tree-2.png)
+
+与ReplicatedMergeTree不同，SharedMergeTree不需要副本之间进行通信。相反，所有通信都通过共享存储和clickhouse-keeper进行。SharedMergeTree实现了异步无领导复制，并使用clickhouse-keeper进行协调和元数据存储。这意味着随着服务的扩展，不需要复制元数据。这可以加快复制、变更、合并和扩展操作。SharedMergeTree允许每个表有数百个副本，使得无需分片即可进行动态扩展。这也意味着在ClickHouse Cloud中，使用分布式查询执行方法可以利用更多的计算资源来执行查询。
+
+
+## 系统监控
+
+用于系统监控的ReplicatedMergeTree的大部分系统表（system table）在SharedMergeTree中也存在，唯独没有system.replication_queue和system.replicated_fetches，因为没有数据和元数据的复制。然而，SharedMergeTree对这两个表有相应的替代表。
```
Comment (hanfei1991) [inline docs/zh/engines/table-engines/mergetree-family/sharedmergetree.md:32]:
```suggestion
用于系统监控的ReplicatedMergeTree的大部分系统表（system table）在SharedMergeTree中也存在，唯独没有`system.replication_queue`和`system.replicated_fetches`，因为没有数据和元数据的复制。然而，SharedMergeTree对这两个表有相应的替代表。
```

File: docs/zh/engines/table-engines/mergetree-family/sharedmergetree.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+```sql
+CREATE TABLE my_table(
+ key UInt64,
+ value String
+)
+ENGINE =MergeTree
+ORDER BY key 
+```
+
+在ClickHouse Cloud中，由于default_table_engine=MergeTree，你不必再特别设置ENGINE=MergeTree。下面的查询语句和上面的完全一样。
```
Comment (hanfei1991) [inline docs/zh/engines/table-engines/mergetree-family/sharedmergetree.md:59]:
```suggestion
在ClickHouse Cloud中，由于 `default_table_engine=MergeTree`，用户不必再特别设置`ENGINE=MergeTree`。下面的查询语句和上面的完全一样。
```

File: docs/zh/engines/table-engines/mergetree-family/sharedmergetree.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+用于系统监控的ReplicatedMergeTree的大部分系统表（system table）在SharedMergeTree中也存在，唯独没有system.replication_queue和system.replicated_fetches，因为没有数据和元数据的复制。然而，SharedMergeTree对这两个表有相应的替代表。
+
+`system.virtual_parts`
+
+这个表作为SharedMergeTree对system.replication_queue的替代，存储关于最新的一组data parts，以及未来正在进行的合并、变更和删除parts。
+
+`system.shared_merge_tree_fetches`
+
+这个表是SharedMergeTree对system.replicated_fetches的替代。它包含关于正在加载入内存的主键和校验码信息。
```
Comment (hanfei1991) [inline docs/zh/engines/table-engines/mergetree-family/sharedmergetree.md:40]:
```suggestion
这个表是SharedMergeTree对`system.replicated_fetches`的替代。它包含关于正在加载入内存的主键和校验码信息。
```

File: docs/zh/engines/table-engines/mergetree-family/sharedmergetree.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+与ReplicatedMergeTree不同，SharedMergeTree不需要副本之间进行通信。相反，所有通信都通过共享存储和clickhouse-keeper进行。SharedMergeTree实现了异步无领导复制，并使用clickhouse-keeper进行协调和元数据存储。这意味着随着服务的扩展，不需要复制元数据。这可以加快复制、变更、合并和扩展操作。SharedMergeTree允许每个表有数百个副本，使得无需分片即可进行动态扩展。这也意味着在ClickHouse Cloud中，使用分布式查询执行方法可以利用更多的计算资源来执行查询。
+
+
+## 系统监控
+
+用于系统监控的ReplicatedMergeTree的大部分系统表（system table）在SharedMergeTree中也存在，唯独没有system.replication_queue和system.replicated_fetches，因为没有数据和元数据的复制。然而，SharedMergeTree对这两个表有相应的替代表。
+
+`system.virtual_parts`
+
+这个表作为SharedMergeTree对system.replication_queue的替代，存储关于最新的一组data parts，以及未来正在进行的合并、变更和删除parts。
```
Comment (hanfei1991) [inline docs/zh/engines/table-engines/mergetree-family/sharedmergetree.md:36]:
```suggestion
这个表作为SharedMergeTree对 `system.replication_queue` 的替代，存储关于最新的一组data parts，以及未来正在进行的合并、变更和删除parts。
```

**Comment by justindeguzman** [comment]
@alexey-milovidov this is good to merge and has been checked by Hanfei, Tracy, Pengyuan, and Jianfei. If you prefer I squash the commits let me know.

**Pull request merged**